internal/k8s/crdcontroller.go

```diff
@@ -809,6 +815,11 @@ func validateHostRuleObj(key string, hostrule *akov1alpha1.HostRule) error {
 // validateMultiClusterIngressObj validates the MCI CRD changes before pushing it to ingestion
 func validateMultiClusterIngressObj(key string, multiClusterIngress *akov1alpha1.MultiClusterIngress) error {
 
+	if !lib.AKOControlConfig().IsLeader() {
```

Comment:
need to change this if conditions.

Reply:
Hi @apalsule @akshayhavile @aaha97,
Added a validator interface as per the discussion. could you please review it?